000-cbdb-sandbox/README.md

```diff
@@ -102,5 +122,59 @@ In addition to using the `docker exec` command, you can also use the `ssh` comma
 ssh gpadmin@localhost # Password: cbdb@123
 ```
 
-Now you have a Cloudberry Database and can continue with [Cloudberry Database Tutorials Based on Single-Node Installation](https://github.com/cloudberrydb/bootcamp/blob/main/101-cbdb-tutorials/README.md)! Enjoy!
+Now you have a Cloudberry Database and can continue with [Cloudberry Database Tutorials Based on Docker Installation](https://github.com/cloudberrydb/bootcamp/blob/main/101-cbdb-tutorials/README.md)! Enjoy!
+
+## Working with your Cloudberry Docker environment
+
+When working with the Cloudberry Docker environment there are a few commands that will be useful to you.
+
+**Stopping Your Single Container Deployment With Docker**
+
+To stop the **single container** deployment while _keeping the data and state_ within the container, you can run the command below. This means that you can later start the container again and any changes you made to the containers will be persisted between runs.
+
+```shell 
+docker stop cbdb-mdw
+```
+
+To stop the **single container** deployment and also remove the volume that belongs to the container, you can run the following command. Keep in mind this will remove the volume as well as the container associated which means any changes you've made inside of the container or any database state will be wiped and unrecoverable.
+
+```shell 
+docker rm -f cbdb-mdw
```

Comment:
I noticed there is no corresponding command for the multi container config. I actually ran this and noticed I had three containers remaining :)

Reply:
I think this was confusion since there weren't the subheadings from your afforementioned comments on this PR. Now that those subheadings are there it should hopefully stop users from running this command and scrolling down to the "Multi-container" deployment subheading. Might make more sense though to split the useful commands section by deployment types instead of user intentions like how it is now (see this comment as well https://github.com/cloudberrydb/bootcamp/pull/33#discussion_r1707659964)